globus_sdk/transfer/client.py

```diff
@@ -1419,8 +1419,8 @@ def task_successful_transfers(self, task_id, num_results=100, **params):
         .. note::
 
             This does not include files that were checked but skipped as part of a sync
-            transfer, only files that were actually transferred, and does not include
-            any directories.
+            transfer or skipped due to skip_source_errors, only files that were
+            actually transferred, and does not include any directories.
```

Comment:
This sentence is now very long; I think we should break it up.

I suggest:
> Only files that were actually transferred are included.
> This does not include files that were checked but skipped as part of a sync
> transfer, files which were skipped due to ``skip_source_errors``, or
> directory names.

But if you have an alternate phrasing you prefer, go for it.

Reply:
Thank you! I slightly adjusted the phrasing to put directories first since that seems the most common case.